core-bundle/contao/library/Contao/Image.php

```diff
@@ -74,7 +74,7 @@ public static function getPath($src)
 
 		$src = rawurldecode($src);
 
-		if (strpos($src, '/') !== false)
+		if (str_contains($src, '/'))
```

Comment:
That was the cs-fixer - Seems to be the same in 5.3 and 5.x already

Reply:
Your dependencies were probably on the `5.3` or `5.x` state. May be you need to run `composer update && composer bin all install` again, before running the CS fixer etc.

Reply:
That did work @fritzmg